webapp/src/app.tsx

```diff
@@ -1445,7 +1445,7 @@ export class ProjectView
         pxt.tickEvent("sandbox.openfulleditor");
         Util.assert(pxt.shell.isSandboxMode());
 
-        let editUrl = pxt.appTarget.appTheme.embedUrl;
+        let editUrl = pxt.appTarget.appTheme.shareUrl;
```

Comment:
Maybe add || homeUrl in case shareUrl is missing.

Looks good - maybe we should give a round of UI Redesign to the share page :)

Reply:
We have shareURL in v0 which only microbit points. This is specific to v0 and I will keep this way for now.